src/cdk/v2/destinations/lytics/procWorkflow.yaml

```diff
@@ -0,0 +1,78 @@
+bindings:
+  - name: EventType
+    path: ../../../../constants
+  - name: flattenJson
+    path: ../../../../v0/util
+  - name: removeUndefinedAndNullValues
+    path: ../../../../v0/util
+
+steps:
+  - name: validateInput
+    template: |
+      $.assert(.message.type, "message Type is not present. Aborting message.");
+      $.assert(.message.type in {{$.EventType.([.PAGE, .TRACK, .SCREEN, .IDENTIFY])}}, 
+        "message type " + .message.type + " is not supported");
+  - name: prepareContext
+    template: |
+      $.context.messageType = .message.type.toLowerCase();
+      $.context.endpoint = "https://api.lytics.io/collect/json/"+.destination.Config.stream +"?access_token="+.destination.Config.apiKey;
+      $.context.payload = {};
+  - name: identifyPayload
```

Comment:
lets use generic paths

Reply:
doing as current implementation, we can do improvements later as separate activity post migration

Reply:
Done